src/Files.Uwp/Views/SettingsPages/Preferences.xaml

```diff
@@ -51,15 +51,27 @@
                 </ComboBox>
             </local:SettingsBlockControl>
 
-            <local:SettingsBlockControl Title="{helpers:ResourceString Name=DateFormat}" HorizontalAlignment="Stretch">
+            <local:SettingsBlockControl Title="{helpers:ResourceString Name=DateFormat}" Description="{Binding DateFormatSample}" HorizontalAlignment="Stretch">
                 <local:SettingsBlockControl.Icon>
                     <FontIcon Glyph="&#xEC92;" />
                 </local:SettingsBlockControl.Icon>
                 <ComboBox
-                    x:Name="DateFormatChooser"
+                    x:Name="DateFormatChooser" Grid.Column="1"
                     AutomationProperties.Name="{helpers:ResourceString Name=DateFormat}"
                     ItemsSource="{Binding DateFormats}"
-                    SelectedIndex="{Binding SelectedDateFormatIndex, Mode=TwoWay}" />
+                    SelectedIndex="{Binding SelectedDateFormatIndex, Mode=TwoWay}">
+                    <ComboBox.ItemTemplate>
+                        <DataTemplate x:DataType="settingsviewmodels:DateFormatItem">
+                            <StackPanel Orientation="Vertical">
+                                <TextBlock Text="{x:Bind Label}" />
+                                <TextBlock Margin="16,0,0,0" FontWeight="Light" FontSize="12"
```

Comment:
What are your thoughts on using a smaller margin?

Reply:
We can reduce it.  I would do that tomorrow.

Reply:
It's done.